cluster/examples/kubernetes/rook-pool.yaml

```diff
@@ -6,9 +6,11 @@ metadata:
 spec:
   # The failure domain will spread the replicas of the data across different failure zones
   failureDomain: osd
+  # The root of the crush hierarchy that will be used for the pool. If not set, will use "default".
+  crushRoot: default
   # For a pool based on raw copies, specify the number of copies. A size of 1 indicates no redundancy.
   replicated:
-    size: 1
+   size: 1
```

Comment:
since size is a property of replicated, shouldn't it still be indented in 2 spaces?

Reply:
yes, did not intend to change that spacing